framework/src/security/security_utils.c

```diff
@@ -267,11 +267,14 @@ int secutils_convert_path_s2h(const char *path, uint32_t *slot)
 	if (!strncmp(path, SS_PATH, sizeof(SS_PATH) - 1)) {
 		*slot = atoi(&path[3]);
 		return 0;
-	} else if (!strncmp(path, FACTORY_PATH, sizeof(FACTORY_PATH) - 1)) {
+	}
+#if (defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053)) \
```

Comment:
Is `(defined(CONFIG_ARCH_BOARD_ARTIK053S) || defined(CONFIG_ARCH_BOARD_ARTIK053))` necessary?
If it is, why not ARTIK055S?

Reply:
FACTORYKEY_ARTIK_CERT is in isp_custom.h and the header file can only be applicable for ARTIK053 series.
Because SE H/W module is only loaded on either ARTIK053 or ARTIK053s..

Reply:
Anyway this "else if" is deleted.